### DISCUSSION TITLE
Is there a way to search all contacts near another contact when viewing it in the map?

### DISCUSSION THREAD

**Discussion by iknowpc** [mithnusa/odoo-google-maps#64] [Q&A]
Usually the map shows 80 contacts at a time, but if you search for a particular contact, it can give you the one you are looking for.  How can it then show all the contacts that are nearby this selected contact?  I haven't found an easy way to do this.

**Comment by gityopie** [comment]
Hi @iknowpc 

Other than showing all contacts, right now there is no way as you described.
One possible way is to have a smart button in contact form (later we can add the button to list contacts in map's sidebar)
